zedd-app/src/components/ClarityView.tsx

```diff
@@ -321,45 +333,49 @@ export const ClarityView = observer((props: ClarityViewProps) => {
           </tr>
         </thead>
         <tbody>
-          {tasksToShow.map((taskToShow) => (
-            <tr
-              key={taskToShow.taskIntId}
-              style={-1 !== taskToShow.taskIntId ? {} : { color: theme.palette.error.main }}
-            >
-              <td>
-                <span style={{ whiteSpace: 'nowrap' }}>{taskToShow.projectName}</span>
-                {' / '}
-                <span style={{ whiteSpace: 'nowrap' }}>{taskToShow.taskName}</span>
-              </td>
-              {intervals.map((w) => {
-                const workEntries = eachDayOfInterval(w)
-                  .flatMap((d) => clarityExport[isoDayStr(d)] ?? [])
-                  ?.filter((we) => we.taskIntId === taskToShow.taskIntId)
-                return (
-                  <td
-                    key={taskToShow.taskIntId + '-' + isoDayStr(w.start)}
-                    title={workEntries.map((we) => we.comment).join('\n')}
-                    style={{ cursor: workEntries.some((we) => we.comment) ? 'help' : 'default' }}
-                    className='numberCell'
-                  >
-                    {workEntries.some((we) => we.comment) && (
-                      <span style={{ fontSize: 'xx-small' }}>m/K </span>
-                    )}
-                    {formatHours(sum(workEntries.map((we) => we.hours)))}
-                  </td>
-                )
-              })}
-              <td className='numberCell'>
-                {formatHours(
-                  sum(
-                    allWorkEntries
-                      .filter((we) => we.taskIntId === taskToShow.taskIntId)
-                      .map((we) => we.hours),
-                  ),
-                )}
-              </td>
-            </tr>
-          ))}
+          {tasksToShow
+            .filter((taskToShow) =>
+              taskToShow.projectName.toLowerCase().includes(valueFilterProject.toLowerCase()),
+            )
```

Comment:
`tasksToShow` should probably already be filtered to those we are showing. Additionally, it makes sense to also filter on `name`, not only `projectName`